exonum_client/client.py

```diff
@@ -233,6 +233,42 @@ def available_services(self) -> requests.Response:
         """
         return _get(_SYSTEM_URL.format(self.schema, self.hostname, self.public_api_port, "services"))
 
+
+    def get_instance_id_by_name(self, name: str) -> int:
+        """
+        Gets an instance id with provided name.
+
+        Example:
+
+        >>> id = client.get_instance_id_by_name("cryptocurrency")
+        >>> id
+        42
+
+        Parameters
+        ----------
+        name: Name of the service.
+
+        Returns
+        -------
+        result: int
+            ID of the instance.
+        """
+        id = None
+        available_services = self.available_services().json()
+        if not "services" in available_services:
```

Comment:
AFAIK "services" are always in the response, even if there are no running services.